src/hardware/serialport/softmodem.cpp

```diff
@@ -1008,8 +1009,26 @@ void CSerialModem::Timer2()
 		}
 	}
 
+	// Discard any incoming traffic while in command mode
+	if (commandmode && clientsocket) {
+		size_t usesize = 80;
```

Comment:
I've done some digging, and looks like that's the max number of bytes to be received in bytes. The name should reflect that (e.g. `constexpr size_t MaxReceiveBytes = 80` or something).

Also, why 80 and not any other number? I think a comment is in order to explain the magic number.

Reply:
I just wanted to pick some big number to make sure we can discard data as quickly as possible.

Reply:
Put this comment in there then:

"The read size doesn't really matter; we just want to discard data quickly enough".